pymc3/progressbar.py

```diff
@@ -54,7 +55,12 @@ def animate(self, i, elapsed):
 
     def progbar(self, i, elapsed):
         bar = self.bar(self.percentage(i))
-        return "[%s] %i of %i complete in %.1f sec" % (bar, i, self.iterations, round(elapsed, 1))
+        sps = i / float(elapsed) 
+        eta = (self.iterations / sps) - elapsed
+        return ("[%s] %i of %i in %.1f sec | SPS: %.1f | ETA: %s" % 
```

Comment:
I think a `.format()` would be nicer here:
`"[{bar}] {iteration} of {total_iterations} in ...".format(bar=bar, ...)`